src/component/Board.jsx

```diff
@@ -2,53 +2,121 @@ import { useState } from "react";
 import Card from "./Card";
 
 const cardValue = [];
-const cardLevel = 6;
+const cardLevel = 8;
 let card = [];
 
 // creating values of the card
 const createCardValue = (cardLevel) => {
   for (let i = 1; i <= cardLevel; i++) {
     cardValue.push(Math.ceil(i / 2));
   }
-  // console.log(cardValue);
 };
 
-createCardValue(cardLevel);
+// shuffle the cardValue array
+function shuffle(array) {
+  array.sort(() => Math.random() - 0.5);
+}
 
 // creating card
 const createCards = (cardValue) => {
   for (let i = 0; i < cardValue.length; i++) {
     card.push({
       flip: false,
       value: cardValue[i],
+      id: i + 1, // id updated
```

Comment:
This change might break other logics can you please check flow again?

Reply:
solve the problem